pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -470,7 +488,7 @@ func (a *ofLearnAction) MatchLearnedDstIPv6() LearnAction {
 }
 
 // MatchReg makes the learned flow to match the data in the reg of specific range.
-func (a *ofLearnAction) MatchReg(regID int, data uint32, rng Range) LearnAction {
+func (a *ofLearnAction) MatchReg(regID int, data uint32, rng *Range) LearnAction {
 	toField := &ofctrl.LearnField{Name: fmt.Sprintf("NXM_NX_REG%d", regID), Start: uint16(rng[0])}
```

Comment:
Do you think it is a good idea to save the string for the predefined regs?

Reply:
I could add "name" as a property in `RegField`, but I don't know how to leverage this property..

Reply:
How about saving the LearnField?

Reply:
I didn't get your point about "saving the LearnField". Here `LearnField` is a type defined in ofnet.